src/js.rs

```diff
@@ -0,0 +1,14 @@
+use crate::{
+    ast::*, error::Error::UnsupportedFeature, fs::Utf8Writer, line_numbers::LineNumbers, Result,
+};
+
+pub fn module(
+    _module: &TypedModule,
+    _line_numbers: &LineNumbers,
+    _writer: &mut impl Utf8Writer,
+) -> Result<()> {
+    Err(UnsupportedFeature {
+        target: crate::Target::JavaScript,
+        feature: "Any feature! ".to_string(),
```

Comment:
Obviously, this message will start saying something more useful from the very next PR